.github/workflows/check-rtw.yml

```diff
@@ -6,11 +6,15 @@ on:
   # Triggers the workflow on push events
   push:
     paths:
-#      - esmvaltool/utils/recipe_test_workflow/**
+      - esmvaltool/utils/recipe_test_workflow/**
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
```

Comment:
this will still trigger at every push to `main` - which is fine, but I thought we're turning that feature off via this PR, since now we run it every 10 days

Reply:
I thought `workflow_dispatch` just enables the workflow to be run manually? https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow and https://github.com/ESMValGroup/ESMValTool/actions/workflows/check-rtw.yml?

Reply:
that's what I thought too! But reading carefully their docs on it https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch - which incidentally is different than your docs entry (GA docs are a mess!), it says there in the table that it'll trigger on the last commit on the `GITHUB_REF` branch, which is `main` in our case; if you look at the Actions we run, you'll see it triggers at every push to the main branch too, so in actuality, it's the equivalent of:
```
on:
  push:
    branches:
      - main
```
which is fine IMHO, since we run a bunch of other more intensive workflows triggered at the same time; but if you want it to run only periodically, then I suggest removing this trigger :beer:

Reply:
Ah, GitHub! Ok, I have removed it 😊 a58d660

Reply:
cheers, Emma! Shall merge now then :beers: